superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -159,6 +162,10 @@ export function Menu({
 }: MenuProps) {
   const [dropdownOpen, setDropdownOpen] = useState(false);
 
+  // would useQueryParam here but not all apps provide a router context
+  const standalone = getUrlParam('standalone', 'boolean');
+  if (standalone) return <></>;
```

Comment:
oh this is cool, we'll support `standalone=true` for all pages

Reply:
In the course of writing this I also learned about `standalone=2` which removes the dashboard header lol

Reply:
we could have `standalone` be a set of bit flags where each bit is a different feature to remove

Reply:
🤦